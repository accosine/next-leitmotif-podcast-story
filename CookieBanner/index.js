import React from "react";
import { AmpUserNotification } from "../Amp";

const CookieBanner = ({
  layout,
  id,
  buttontext,
  children,
  className,
  buttonClassName,
}) => {
  return (
    <AmpUserNotification
      layout={layout}
      id={id}
      buttontext={buttontext}
      className={className}
      buttonClassName={buttonClassName}
    >
      {children}
    </AmpUserNotification>
  );
};

export default CookieBanner;
