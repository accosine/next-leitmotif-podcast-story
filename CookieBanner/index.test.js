import React from "react";
import "@testing-library/jest-dom/extend-expect";
import { render } from "@testing-library/react";
import CookieBanner from "./index";

describe("Attachment", () => {
  it("renders the CookieBanner component", () => {
    const { container } = render(<CookieBanner />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
