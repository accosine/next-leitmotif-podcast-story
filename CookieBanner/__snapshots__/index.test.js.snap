// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Attachment renders the CookieBanner component 1`] = `
<amp-user-notification
  classname=""
  id="amp-user-notification1"
  layout="nodisplay"
>
  <button
    class=""
    on="tap:amp-user-notification1.dismiss"
  >
    Yay cookies, nom nom nom
  </button>
</amp-user-notification>
`;
